app/domain/errors.rb

```diff
@@ -353,4 +353,18 @@ module Util
       code: "CONJ00044E"
     )
   end
+
+  module System
+    EmptyConjurDataKey = ::Util::TrackableErrorClass.new(
+      msg:  "CONJUR_DATA_KEY is empty. " \
+            "Try loading the master key as an environment variable.",
```

Comment:
@eranha  (cc: @rafis3 ) Just want to make sure that this "master key" is the key from the encryption of the server keys....

Reply:
@shulifink good catch! No, the master key is used in DAP to encrypt the server keys. This key, which is the data key, is one of the sever keys that the master key encrypts. So we should rewrite this message and explicitly call it data key.